src/nvim/syntax.c

```diff
@@ -3421,9 +3421,7 @@ static void syn_cmd_on(exarg_T *eap, int syncing)
  */
 static void syn_cmd_enable(exarg_T *eap, int syncing)
```

Comment:
as this is a static function, you can just as well delete it and use `syn_cmd_on` where it is mentioned (two places)